app/src/main/java/nerd/tuxmobil/fahrplan/congress/designsystem/typography/Typography.kt

```diff
@@ -0,0 +1,53 @@
+package nerd.tuxmobil.fahrplan.congress.designsystem.typography
+
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.material3.Typography as Material3Typography
+
+@Immutable
+data class Typography(
+    // Material 3
+    val displayLarge: TextStyle,
+    val displayMedium: TextStyle,
+    val displaySmall: TextStyle,
+    val headlineLarge: TextStyle,
+    val headlineMedium: TextStyle,
+    val headlineSmall: TextStyle,
+    val titleLarge: TextStyle,
+    val titleMedium: TextStyle,
+    val titleSmall: TextStyle,
+    val bodyLarge: TextStyle,
+    val bodyMedium: TextStyle,
+    val bodySmall: TextStyle,
+    val labelLarge: TextStyle,
+    val labelMedium: TextStyle,
+    val labelSmall: TextStyle,
+
+    // App-specific
+    val preferenceTitle: TextStyle,
+)
+
+fun Typography.toMaterial3Typography(): Material3Typography {
+    return Material3Typography(
+        displayLarge = displayLarge,
+        displayMedium = displayMedium,
+        displaySmall = displaySmall,
+        headlineLarge = headlineLarge,
+        headlineMedium = headlineMedium,
+        headlineSmall = headlineSmall,
+        titleLarge = titleLarge,
+        titleMedium = titleMedium,
+        titleSmall = titleSmall,
+        bodyLarge = bodyLarge,
+        bodyMedium = bodyMedium,
+        bodySmall = bodySmall,
+        labelLarge = labelLarge,
+        labelMedium = labelMedium,
+        labelSmall = labelSmall,
+    )
+}
+
+internal val LocalTypography = staticCompositionLocalOf<Typography> {
```

Comment:
What is a good consistent place for these `staticCompositionLocalOf`. It would be nice to follow the same pattern for both `LocalColorScheme` and `LocalTypography`.

Reply:
This feels like a good place. `LocalColorScheme` could be moved from `ColorSchemes.kt` to `ColorScheme.kt`. But that should probably be a separate PR.

Reply:
OK. Sounds good. I took inspiration from [here](https://github.com/thunderbird/thunderbird-android/blob/main/core/ui/compose/theme2/common/src/main/kotlin/app/k9mail/core/ui/compose/theme2/ThemeColorScheme.kt) anyway.